database/migrations/2020_09_06_061434_create_projects_table.php

```diff
@@ -15,7 +15,8 @@ public function up()
 			$table->integer('status_id')->unsigned();
 			$table->string('repo_link', 255)->nullable();
 			$table->smallInteger('max_member_count')->default('1');
-			$table->date('deadline')->nullable();
+            $table->date('startdate')->nullable();
+			$table->date('enddate')->nullable();
```

Comment:
Could you add this in a new migration as well? It'd make it easier to update the server.